.travis.yml

```diff
@@ -16,11 +16,15 @@ matrix:
     - php: 7.2
       env:
         - COMPOSER_FLAGS="--prefer-lowest --prefer-dist --no-interaction"
+        - APP_DEBUG=1
+        - APP_ENV=test
     - php: 7.3
```

Comment:
Would test on `7.4` and use pcov for code coverage: see changes here: https://github.com/sulu/SuluCommunityBundle/pull/127/files#diff-354f30a63fb0907d4ad57269548329e3